[2024-01-09T17:12:24.213+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task scheduled__2023-12-02T07:00:00+00:00 [queued]>
[2024-01-09T17:12:24.246+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task scheduled__2023-12-02T07:00:00+00:00 [queued]>
[2024-01-09T17:12:24.248+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 2
[2024-01-09T17:12:24.298+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2023-12-02 07:00:00+00:00
[2024-01-09T17:12:24.321+0000] {standard_task_runner.py:60} INFO - Started process 84 to run task
[2024-01-09T17:12:24.335+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag', 'local_to_gcs_task', 'scheduled__2023-12-02T07:00:00+00:00', '--job-id', '97', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcp.py', '--cfg-path', '/tmp/tmpn3cy63is']
[2024-01-09T17:12:24.350+0000] {standard_task_runner.py:88} INFO - Job 97: Subtask local_to_gcs_task
[2024-01-09T17:12:24.625+0000] {task_command.py:423} INFO - Running <TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task scheduled__2023-12-02T07:00:00+00:00 [running]> on host f5de67cb9685
[2024-01-09T17:12:24.858+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/context.py:207: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2024-01-09T17:12:25.016+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_ingestion_gcs_dag' AIRFLOW_CTX_TASK_ID='local_to_gcs_task' AIRFLOW_CTX_EXECUTION_DATE='2023-12-02T07:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-02T07:00:00+00:00'
[2024-01-09T17:12:25.705+0000] {taskinstance.py:2699} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 199, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 216, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/ingest_data_gcp_script.py", line 21, in upload_to_gcs
    with storage.Client() as client:
AttributeError: __enter__
[2024-01-09T17:12:25.744+0000] {taskinstance.py:1138} INFO - Marking task as UP_FOR_RETRY. dag_id=data_ingestion_gcs_dag, task_id=local_to_gcs_task, execution_date=20231202T070000, start_date=20240109T171224, end_date=20240109T171225
[2024-01-09T17:12:25.920+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 97 for task local_to_gcs_task (__enter__; 84)
[2024-01-09T17:12:25.951+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-01-09T17:12:26.050+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-01-10T00:13:14.037+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task scheduled__2023-12-02T07:00:00+00:00 [queued]>
[2024-01-10T00:13:14.090+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task scheduled__2023-12-02T07:00:00+00:00 [queued]>
[2024-01-10T00:13:14.090+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2024-01-10T00:13:14.114+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2023-12-02 07:00:00+00:00
[2024-01-10T00:13:14.126+0000] {standard_task_runner.py:60} INFO - Started process 6789 to run task
[2024-01-10T00:13:14.144+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag', 'local_to_gcs_task', 'scheduled__2023-12-02T07:00:00+00:00', '--job-id', '137', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcp.py', '--cfg-path', '/tmp/tmp_grmnzj8']
[2024-01-10T00:13:14.157+0000] {standard_task_runner.py:88} INFO - Job 137: Subtask local_to_gcs_task
[2024-01-10T00:13:14.255+0000] {task_command.py:423} INFO - Running <TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task scheduled__2023-12-02T07:00:00+00:00 [running]> on host f5de67cb9685
[2024-01-10T00:13:14.325+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/context.py:207: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2024-01-10T00:13:14.383+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_ingestion_gcs_dag' AIRFLOW_CTX_TASK_ID='local_to_gcs_task' AIRFLOW_CTX_EXECUTION_DATE='2023-12-02T07:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-02T07:00:00+00:00'
[2024-01-10T00:13:14.518+0000] {ingest_data_gcp_script.py:24} INFO - Starting upload of /opt/***/input_2023-12.parquet to raw/yellow_tripdata_2023_12.parquet
[2024-01-10T00:13:15.006+0000] {ingest_data_gcp_script.py:29} INFO - Finished upload of /opt/***/input_2023-12.parquet to raw/yellow_tripdata_2023_12.parquet
[2024-01-10T00:13:15.007+0000] {python.py:201} INFO - Done. Returned value was: https://storage.googleapis.com/dtc-de-project1-terra-bucket/raw/yellow_tripdata_2023_12.parquet
[2024-01-10T00:13:15.039+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag, task_id=local_to_gcs_task, execution_date=20231202T070000, start_date=20240110T001314, end_date=20240110T001315
[2024-01-10T00:13:15.080+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-10T00:13:15.129+0000] {taskinstance.py:3281} INFO - 1 downstream tasks scheduled from follow-on schedule check
